plugin.py

```diff
@@ -18,3 +19,15 @@ class LspHtmlPlugin(NpmClientHandler):
     @classmethod
     def install_in_cache(cls) -> bool:
```

Comment:
Can you remove this override? Since the default in `lsp_utils` was toggled, we can now remove those overrides on whenever we have a chance.